cli/dts/lib.webworker.d.ts

```diff
@@ -695,7 +723,7 @@ interface AbortController {
     /** Returns the AbortSignal object associated with this object. */
     readonly signal: AbortSignal;
     /** Invoking this method will set this object's AbortSignal's aborted flag and signal to any observers that the associated activity is to be aborted. */
-    abort(reason?: any): void;
+    // abort(): AbortSignal; - To be re-added in the future
```

Comment:
Is this on purpose? Looks strange

Reply:
Agreed, this seems a bit problematic.

Reply:
It is what is upstream: https://github.com/microsoft/TypeScript/blob/release-4.7/lib/lib.webworker.d.ts#L726

I really don't want to get into updating their distributables if we can avoid it.  It appears they have fixed it in main, but that is what they issued with 4.7.  The impact on Deno users will be minimal, because it only gets used when users explicitly use it.